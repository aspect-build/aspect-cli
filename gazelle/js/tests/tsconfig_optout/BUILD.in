# gazelle:js_tsconfig enabled
# gazelle:js_tsconfig_ignore allow_js
