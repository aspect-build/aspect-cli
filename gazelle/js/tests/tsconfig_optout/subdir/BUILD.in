# gazelle:js_tsconfig_ignore resolve_json_module
# gazelle:js_tsconfig_ignore declaration_map
# gazelle:js_tsconfig_ignore allow_js
