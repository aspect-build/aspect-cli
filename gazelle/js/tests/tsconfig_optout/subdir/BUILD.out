load("@aspect_rules_ts//ts:defs.bzl", "ts_config", "ts_project")

# gazelle:js_tsconfig_ignore resolve_json_module
# gazelle:js_tsconfig_ignore declaration_map
# gazelle:js_tsconfig_ignore allow_js

ts_project(
    name = "subdir",
    srcs = ["main.ts"],
    composite = True,
    declaration = True,
    incremental = True,
    source_map = False,
    ts_build_info_file = "dist/tsconfig.tsbuildinfo",
)

ts_config(
    name = "tsconfig",
    src = "tsconfig.json",
    deps = ["//:tsconfig"],
)
